docs/xcm/building-with-xcm/xc-reserve-transfer.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 7
+---
+
+# Reserve Transfer Assets
+
+XCM allowus us to transfer assets from one chain to another in several ways. In this chapter we'll see how to use reserve transfers.
+
+First of all, you need to understand that we don't just magically teleport an asset to a remote chain, i.e. actual tokens do not leave the local chain, they just migrate to a special local account controlled by a remote entity. Basically we create a derivative that will represent local asset on a remote chain. This is done by registering an asset and mapping it's supply to a sovereign account. That way, the `assets` pallet will handle token minting and burning automatically, so the amount of minted derivatives will always match amount of actual tokens residing in corresponding sovereign account.
```

Comment:
There are a couple of things that are incorrect here.

It's not us that creates a derivative, it's the destination chain. We don't have control over how they do this. There's no guarantee they're using `pallet-assets` at all or how they handle anything in that regard.

There is no actual mapping between an asset and a sovereign account. The concept of sovereign account is to have a privileged representation of a sibling parachain or relay chain locally in the form of an account.
The way it works is that when you use `reserve_transfer`, we can derive the destination parachain's sovereign account. But this isn't connected with an asset Id in any way.

Imo, this paragraph should either be rewritten or dropped.

Reply:
> We don't have control over how they do this. There's no guarantee they're using pallet-assets at all or how they handle anything in that regard.

Good point. My original intention was to provide a context for the reader, i.e. what _could_ (but not necessarily _will_) happen. Thank you for the feedback.

Looks like currently I just mislead the reader.